lib/tabViewController/TabViewController.dart

```diff
@@ -37,79 +38,78 @@ class _TabViewControllerState extends State<TabViewController> {
     final double height = MediaQuery.of(context).size.height * 0.52;
 
     return Scaffold(
-        body: Stack(
-            children: <Widget>[
-              ClipPath(child:
-              Container(color: Constants.PURPLE, height: height), clipper: BezierClipper()),
-              Column(
-                  children: [
-                    const Padding(padding: EdgeInsets.only(top: 50.0)),
-                    Stack(
-                      children: [Row(
-                        children:
-                        [const Padding(padding: EdgeInsets.only(right: 30.0)),
-                          _buildLogos("HELP"),
-                          const Padding(padding: EdgeInsets.only(right: 80.0)),
-                          loadHalfLogo(),
-                          const Padding(padding: EdgeInsets.only(right: 70.0)),
-                          _buildLogos("BELL")],
+        body: SafeArea(
+          child: Stack(
+              children: <Widget>[
+                ClipPath(child:
+                Container(color: Constants.PURPLE, height: height), clipper: BezierClipper()),
+                Column(
+                    children: [
+                      //const Padding(padding: EdgeInsets.only(top: 25.0)),
+                      Stack(
+                          children: [Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceAround,
+                            children:
+                            [//const Padding(padding: EdgeInsets.only(right: 30.0)),
+                              _buildLogos("HELP"),
+                              //const Padding(padding: EdgeInsets.only(right: 80.0)),
```

Comment:
delete

Reply:
Done in [0c1c603a](https://github.com/andretmarques/UniHub/pull/10/commits/0c1c603a26032b2e8b90cc385a3a5289c4272f69)